Rubberduck.VBEditor.VB6/SafeComWrappers/VB/VBControl.cs

```diff
@@ -0,0 +1,34 @@
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using VB = Microsoft.Vbe.Interop.VB6;
+
+namespace Rubberduck.VBEditor.SafeComWrappers.VB6
```

Comment:
The namespace doesn't match the folder -- folder is `VB`. Shouldn't it be `VB6`?

Reply:
Interesting point. There is a related discussion in Framework Design Guidelines (Cwalina and Abrams) on namespaces and folders. I don't have my copy to hand atm, but IIRC it makes the point that namespaces are primarily for type discoverability, whereas folders are for file organisation. They need not follow each other exactly, and typically the file structure would be more subdivided than the namespace.

Happy to take comments though!

Reply:
Discussed and agreed in SE - added R# directives to suppress inspection warnings.